x-pack/plugins/security_solution/public/management/pages/policy/view/host_isolation_exceptions/components/list.tsx

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiSpacer, EuiText } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { FoundExceptionListItemSchema } from '@kbn/securitysolution-io-ts-list-types';
+import React, { useCallback, useMemo, useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import {
+  MANAGEMENT_DEFAULT_PAGE_SIZE,
+  MANAGEMENT_PAGE_SIZE_OPTIONS,
+} from '../../../../../common/constants';
+import { getPolicyHostIsolationExceptionsPath } from '../../../../../common/routing';
+import {
+  ArtifactCardGrid,
+  ArtifactCardGridProps,
+} from '../../../../../components/artifact_card_grid';
+import { useEndpointPoliciesToArtifactPolicies } from '../../../../../components/artifact_entry_card/hooks/use_endpoint_policies_to_artifact_policies';
+import { SearchExceptions } from '../../../../../components/search_exceptions';
+import { useGetEndpointSpecificPolicies } from '../../../../../services/policies/hooks';
+import { getCurrentArtifactsLocation } from '../../../store/policy_details/selectors';
+import { usePolicyDetailsSelector } from '../../policy_hooks';
+
+export const PolicyHostIsolationExceptionsList = ({
+  exceptions,
+  policyId,
+}: {
+  exceptions: FoundExceptionListItemSchema;
+  policyId: string;
+}) => {
+  const history = useHistory();
+  // load the list of policies>
+  const policiesRequest = useGetEndpointSpecificPolicies();
+  const urlParams = usePolicyDetailsSelector(getCurrentArtifactsLocation);
+
+  const [expandedItemsMap, setExpandedItemsMap] = useState<Map<string, boolean>>(new Map());
+
+  const pagination = {
```

Comment:
Suggestion:memoize this in order to prevent unnecessary component re-renders due to shallow diffs